portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,52 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.Gson;
+import java.io.*;
```

Comment:
Consider not using folded imports as it brings the confusion on what all is imported and also might contain unused imports.

Reply:
Depending on what IDE you're using  I think there is a setting that stops auto folding imports. I can confirm IntelliJ has it.

Reply:
alright